File: 5.Constructor/index_constructor.java
class User {
    String name;
    Int age;
    User(String name, Int age){
        this.name = name;
        this.age = age;
    }
    void greet(){
        System.out.println("Hello " + this.name)
    }
}

User user = new User("John", 25);
user.greet();

File: 5.Constructor/index_constructor.js
// Without Constructor
class User {
  name;
  greet() {
    console.log(`Hello ${this.name}`);
  }
}

const user = new User();
user.name = "John";
user.greet();

// With constructor
class User1 {
  constructor(name, age) {
    this.name = name;
    this.age = age;
  }
  greet() {
    console.log(`Hello ${this.name}`);
  }
}

const user1 = new User1("John", 25);
user1.greet();
